### DISCUSSION TITLE
Unable to get mouse keys working?

### DISCUSSION THREAD

**Discussion by Just-Insane** [manna-harbour/miryoku#170] [Q&A] [answered]
I'm trying to get mouse keys working via the instructions here for ZMK: https://github.com/manna-harbour/miryoku_zmk#mousekeys-pr

Per the instructions, I have the following settings (via Build Inputs):

```
  set-output "board" "nice_nano_v2"
  set-output "shield" "kyria_rev2_left,kyria_rev2_right"
  set-output "alphas" "QWERTY"
  set-output "nav" "default"
  set-output "clipboard" "default"
  set-output "layers" "default"
  set-output "mapping" "default"
  set-output "custom_config" "#define MIRYOKU_KLUDGE_MOUSEKEYSPR"
  set-output "kconfig" "CONFIG_ZMK_DISPLAY=y\nCONFIG_ZMK_RGB_UNDERGLOW=y\nCONFIG_WS2812_STRIP=y\nCONFIG_ZMK_MOUSE=y\nCONFIG_ZMK_SLEEP=y"
  set-output "branches" "zmkfirmware/zmk/main ftc/zmk/mouse-ftc"
```

Doubt it matters, but I am also using [this branch](https://github.com/Just-Insane/miryoku_zmk/tree/custom_layer).

When holding down the mouse layer key and using the mouse keys mentioned here, nothing seems to happen? The cursor does not move, nor do pages with scroll bars move. The R/L/M buttons also don't do anything.

I'm probably missing something basic, but I haven't been able to figure it out.

**Comment by manna-harbour** [comment, marked as answer]
That looks fine. Have you tried over USB? With BT you may need to re-pair as the old descriptor (without mouse) can be cached on the host.

**Reply by Just-Insane** [reply]
Ah, that was it! It makes sense that it would be cached in the BT profile.
